### DISCUSSION TITLE
Being forced to log in with an account, under penalty to have an ugly red banner in overlay, is not very FOSS-friendly

### DISCUSSION THREAD

**Discussion by ferrum** [Kong/insomnia#6962] [Ideas]
I was sincerely negatively surprised that an open source software, one of the few real alternative to Postman, forces its users to log in using a Google or other non-privacy respecting service's account. If we decline, an ugly banner in overlay asks us to log in. I find this policy really against its users and I'm personally moving away from Insomnia, but I'd happily come back if the policy changes. Best regards.

Edit: After reading some discussions, in particular https://github.com/Kong/insomnia/discussions/6565 https://github.com/Kong/insomnia/discussions/6680 seems like @subnetmarco does not plan to reintroduce the old policy for a supposedly "better experience". I am not usually entitled to open source software, but here a lot of contributions were made and the CTO does not give a flying duck and doomed this amazing project to death.

**Comment by subnetmarco** [comment]
> forces its users to log in using a Google or other non-privacy respecting service's account

You can login with methods other than Google, like a plain email address, if you want.

> but here a lot of contributions were made and the CTO does not give a flying duck and doomed this amazing project to death.

Since we made this change things have been better for Insomnia, we are as a matter of fact doubling the team this year. We are resolving many long-lasting issues and addressing the most requested features in the coming months (pre-request scripting, multi-tab, etc), and we are accelerating pretty much every development activity on the product. So, quite frankly, your assumptions are not being reflected in the actual metrics.
